code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -617,7 +617,7 @@
 	cut_overlays()
 	SSvis_overlays.remove_vis_overlay(src, managed_vis_overlays)
 	icon_state = module.cyborg_base_icon
-	if(stat != DEAD && !(IsUnconscious() || IsStun() || IsParalyzed() || low_power_mode)) //Not dead, not stunned.
+	if(stat != DEAD && !(HAS_TRAIT(src, TRAIT_KNOCKEDOUT) || IsStun() || IsParalyzed() || low_power_mode)) //Not dead, not stunned.
```

Comment:
This should just check stat rather than stun, paralysis and the trait (All things which just make you unconcious in stat)

Reply:
fixed in 4fc03d8